### PR TITLE
Move the transactional method `createReport` into another class

### PR DESCRIPTION
**Pull request by markpollack**
* Move helper method for packageMetadata lookup into PackageMetadataRepository

Fixes GH-273

### CODE REVIEW

File: spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/Info.java [hunk 1]
```diff
@@ -84,4 +84,21 @@ public String getDescription() {
...
 		this.description = description;
 	}
+
+	/**
+	 * Create a new Info instance with the given description, first deployed and last deployed dates
+	 * set to the current date-time, and StatusCode.UNKNOWN.
+	 * @param description a string describing the general info status at a level finer than StatusCode
+	 * @return a new Info object
+	 */
+	public static Info createNewInfo(String description) {
```
Comment (jvalkeal) [inline spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/Info.java]: just comment for future as we probably want to overhaul other pojos as well for static methods and think if we want to add Builder interfaces. Naming will easily get off the rails.

**Comment by jvalkeal** [comment]
lgtm, hopefully merging in about 18mins.

File: spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/Info.java (hunk 1 above)
Comment (markpollack) [reply spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/Info.java]: Please create an issue to add the builder pattern here....

Comment (markpollack) [reply spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/Info.java]: So far, I think this is the only case, but we do have a few other sections with this type of 'boilerplate' code which could be moved down into the domain objects....

**Pull request merged**